lib/driver/pg.js

```diff
@@ -290,8 +290,21 @@ var PgDriver = Base.extend({
         } else {
           sql = util.format('ALTER TABLE "%s" ALTER COLUMN "%s" DROP DEFAULT', tableName, columnName);
         }
+        return this.runSql(sql).nodeify(setType.bind(this));
```

Comment:
I actually kept with convention from line 269 which calls the next function in the chain of `changeColumn` functions. Would something like this work better?

     function setDefaultValue(err) {
        ...
        return this.runSql(sql).then(
          setType.bind(this)
        ).nodeify(callback);
     }

     function setType() {
        if (columnSpec.type !== undefined){
          ...
          return this.runSql(sql);
        }
      }